tasks/main.yml

```diff
@@ -49,24 +60,34 @@
 - name: "create the {{ ler53_account_key_dir }} directory"
   file:
     path: "{{ ler53_account_key_dir }}"
+    state: directory
     owner: root
     mode: 0700
-    state: directory
 
-- include: virtualenv.yml
+- name: "importing virtualenv tasks"
+  import_tasks: virtualenv.yml
   when: ansible_os_family == 'RedHat'
 
+- name: "set hostname to {{ ler53_cert_common_name }}"
```

Comment:
Could you please remove this task? When requesting a Lets Encrypt certificate, I would not expect my system's hostname to change, especially when that system is only used to generate the certificate for something like a network device.

Reply:
Hum, you're right. Having a matching hostname would be a good feature in some cases but yes, it's not a mandatory part of setting up the certificate. Removing it.